### PR TITLE
Update template.css

### PR DESCRIPTION
**Pull request by pau1a**
Pull Request for Issue #25611 .

### Summary of Changes

Added css to separate buttons as per Issue #25611. Currently unsure about any extant convention for positioning within file. Have tried to avoid the obvious elements that should be near the top of the file. 

### Testing Instructions

1. Login to joomla admin panel
2. Go to content -> media
3. Click on create new folder

### Expected result

fields should have some space

### Actual result

fields should have some space

### Documentation Changes Required

None

### CODE REVIEW

**Comment by C-Lodder** [comment]
You need to update the LESS files and compile

**Comment by brianteeman** [comment]
See this doc for instructions https://docs.joomla.org/Joomla_LESS/en

**Comment by pau1a** [comment]
Ah I had my suspicions about that but couldn't be sure. Will amend.

**Comment by C-Lodder** [comment]
Also, a general margin will add 10px of space in each direct. Instead I'd suggest adding a bottom margin to the parent container

**Comment by pau1a** [comment]
How about a left/right margin of 2px as the buttons do seem to be right up against each other also.

**Comment by pau1a** [comment]
Propose to place these changes into /administrator/templates/isis/less/blocks/_media.less

File: administrator/templates/isis/less/blocks/_media.less [hunk 1]
```diff
@@ -1,5 +1,12 @@
 // Media 
 
+/* Spacing out buttons in media manager */
+#j-main-container .media .btn{
+   margin-bottom:5px;
```
Comment (Quy) [inline administrator/templates/isis/less/blocks/_media.less]: Add space after `:`. Repeat next line.

File: administrator/templates/isis/less/blocks/_media.less [hunk 2]
```diff
@@ -1,5 +1,12 @@
 // Media 
 
+/* Spacing out buttons in media manager */
+#j-main-container .media .btn{
+   margin-bottom: 5px;
```
Comment (Quy) [inline administrator/templates/isis/less/blocks/_media.less]: Use tab to indent.

**Comment by gogicomputers** [comment]
I have tested this item :white_check_mark: successfully on 70739244e0b96564ade1a8052a3af39c5f90dc7e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25629">issues.joomla.org/tracker/joomla-cms/25629</a>.</sub>

**Comment by ghost** [comment]
@uthorat please test as opener of #25611.

How to: https://docs.joomla.org/Testing_Joomla!_patches

**Comment by N6REJ** [comment]
I would recommend instead of using 2px to use .25em instead.  This is eqv to a "space" char, and being that its fluid will scale to other changes.

**Comment by brianteeman** [comment]
It is fine as px. Everything else in this template is in px. Just changing this one value to em wont have the benefit you expect - you would need to convert everything

**Comment by SharkyKZ** [comment]
This breaks button group display. Margin should be applied to the form, not to the buttons. Maybe apply custom class to folder form, like `.uploadform` is applied to the upload form right above the folder form?

**Comment by pau1a** [comment]
When you say button group do you mean the fact that the buttons are paired and touching like elsewhere in the backend for yes/no buttons etc? 
If we agree that these buttons should be treated in the same way then Id think just removing the margin-right will get that job done..

**Comment by N6REJ** [comment]
@brianteeman actually thats not true, we specifically looked at both with the icons.  They were changed from px to em ( specifically a single space ) which solved the issues.  The point I think you missed is, .25em = char $32 which 2px does not.

**Comment by brianteeman** [comment]
em is a relative measure
px is an absolute measure

Its not me that missed something at all

**Comment by N6REJ** [comment]
@brianteeman before you continue to spout on about what you clearly don't comprehend please go get educated.  Your demeaning me on a public platform when you haven't even taken the time to familiarize yourself ACCURATELY with my statements.

As you pointed out a px is a finite measurEment  COMPLETELY unrelated to the content and wholy related to the device.  Ergo 2px on any given display may/may not be of different size.
em is a relative unit.  The "space" character most often used on websites is roughly eqv to 2.5px.  This is not possible for a ton of reasons so you must look to using some other form of measurement ( of which there are many ) 
turns out ( as we found out doing the icon fixing ) that .25em is the perfect size to emulate a "space" character.
Since this is a fluid display that we're working on we MUST move away from fixed sizing like was erroneously done decades ago.
With such a vast array of displays that we now have we have no clue what they will be used/viewed on.  Ergo the push to responsive web design.
Your statement that "this is a px template" is prima facie true it is also false.  It's a bootstrap based template which is a FLUID template!  Because of convention it uses .px but its behavior's are primarily controlled by %'s & em's.
Lastly just because a method has been used in the past doesn't make it the best way.
But who am I to want to actually REMOVE a bug instead of trading one for another

edit: Just incase anyone doubts my statement he's the padding actually at work as it has been for a long time 
image

**Comment by brianteeman** [comment]
If writing that makes you happy then fine. It's factually and technically incorrect and based on many assumptions and misunderstandings, not least a failure to understand my original comment. I suggest you educate yourself better on what an em unit is

**Comment by N6REJ** [comment]
Let's see how well things work with px vs em...
( em )
image
(px)
image
Clearly there are issues.

**Comment by N6REJ** [comment]
@pau1a while your fixing the spacing would you please fix the .29px shift between field and button, and the .29px padding offset in button text.
image

**Comment by brianteeman** [comment]
Your calculation to convert px to em is not correct which is why you see the difference

image

**Comment by N6REJ** [comment]
I never said 2px = .25em, I said .25em = chr$32! .25em is closer to 3px then 2px lol

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on dccdd41813567d49c1e254d7b06df65c58b99ee3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25629">issues.joomla.org/tracker/joomla-cms/25629</a>.</sub>

**Comment by ghost** [comment]
@gogicomputers can you please retest?

File: administrator/templates/isis/less/blocks/_media.less [hunk 3]
```diff
@@ -1,5 +1,10 @@
 // Media 
 
+/* Spacing below buttons in media manager */
+.ventral-space{
+	margin-bottom:5px;
```
Comment (infograf768) [inline administrator/templates/isis/less/blocks/_media.less]: small cs:
should be `margin-bottom: 5px;`

**Comment by gogicomputers** [comment]
I have tested this item :white_check_mark: successfully on 2e041f31b3eec20dfcdaa0191bad2ddac61c5c39<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25629">issues.joomla.org/tracker/joomla-cms/25629</a>.</sub>

**Comment by ghost** [comment]
@infograf768 does this need a second retest?

**Comment by infograf768** [comment]
not necessary. can be set rtc

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by HLeithner** [comment]
Thank you for making Joomla! more pretty.

**Pull request merged**